src/nisarqa/utils/typing.py

```diff
@@ -1,7 +1,15 @@
 from __future__ import annotations
 
+__all__ = [
+    "RunConfigScalar",
```

Comment:
Do we need to add `RootParamGroupT` to `__all__` as well so that we can import it as `nisarqa.RootParamGroupT`?

https://github.com/isce-framework/nisarqa/blob/0a307491b5c5d5ccbdad0601ca9c6856918bb9a4/src/nisarqa/parameters/nisar_params.py#L1538